packages/gatsby-source-wordpress/src/gatsby-node.ts

```diff
@@ -9,10 +9,16 @@ exports[pluginInitApiName] = runApiSteps(
     steps.setErrorMap,
     steps.tempPreventMultipleInstances,
     steps.setRequestHeaders,
+    steps.hideAuthPluginOptions,
   ],
   pluginInitApiName
 )
 
+exports.onPreBootstrap = runApiSteps(
+  [steps.restoreAuthPluginOptions],
+  `onPreBootstrap`
+)
+
```

Comment:
this runs after we write out browser plugin options and before bundling engines, so good spot to do restoration